Kernel/Arch/x86_64/ISABus/HID/PS2KeyboardDevice.cpp

```diff
@@ -23,89 +21,46 @@ void PS2KeyboardDevice::irq_handle_byte_read(u8 byte)
 {
     u8 ch = byte & 0x7f;
     bool pressed = !(byte & 0x80);
-
-    m_entropy_source.add_random_event(byte);
-
```

Comment:
You remove these `add_random_event` invocations without mention in the commit message - it seems useful to take user input into account for entropy sources, so we either need to replace this with something equivalent or explain why we're removing this.

Reply:
Oh I forgot about those! I will hook it in the generic `KeyboardDevice` and `MouseDevice` classes :)

Reply:
I hooked this into the generic HID devices, which should be enough :)